lib/config_file.js

```diff
@@ -3,6 +3,7 @@ var readFileSync = require('fs').readFileSync;
 
 var configFiles = [ '.kibana-plugin-helpers.json', '.kibana-plugin-helpers.dev.json' ];
 var configCache = {};
+var KIBANA_ROOT_OVERRIDE = process.env.KIBANA_ROOT ? resolve(process.env.KIBANA_ROOT) : null;
```

Comment:
What's the advantage of adding an env override here? What about the json config doesn't fit the use case you have here?